modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/JMSResourceReleaserTestCase.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.classloader;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.mule.maven.client.api.MavenClient;
+import org.mule.maven.client.api.MavenClientProvider;
+import org.mule.maven.client.api.model.BundleDependency;
+import org.mule.maven.client.api.model.BundleDescriptor;
+import org.mule.maven.client.api.model.MavenConfiguration;
+import org.mule.runtime.module.artifact.api.classloader.ClassLoaderLookupPolicy;
```

Comment:
The import order is:
import static mule
import static java
import static others

import mule
import java
import others

You can read more about that in: https://docs.google.com/document/d/1N2Axq3ABJjYmj9BjfYQkvJfF0odHQnlb8MAdVQydOWI/edit#heading=h.wspx9u93jab2